data/overview.yaml

```
@@ -63,3 +63,448 @@ linear algebra:
+    Principle of isolated zeros: 
+    Cauchy formulas: 
+    Analyticity of a holomorphic function: 
+    Principle of analytic continuation: 
+    Maximum principle:
+    Isolated singularities: 
+    Laurent series: 
+    Meromorphic functions: 
+    Residue theorem:
+    Sequences and series of holomorphic functions: 
+    Holomorphic stability by uniform convergence:
+
+Topology:
+  Topology and Metric Spaces:
+    Topology of a metric space: 
+    Induced topology: 
+    Finite product of metric spaces:
+    Series: 
+    Limit/cluster/accumulation points: 
+    Limits: 
+    Continuous functions: 
+    Homeomorphisms:
+    Compactness: 
+    Equivalence of definitions in terms of cluster points (Bolzano-Weierstrass) or open covers (Borel-Lebesgue): 
+    Connectedness: 
+    Connected components: 
+    Arc connectedness:
+    Lipschitz functions: 
+    uniformly continuous functions: 
+    Heine-Cantor theorem:
+    Complete metric spaces: 
+    Fixed point theorem for continuous functions:
+  Normed vector spaces on R and C:
+    Topology on a normed vector space: 
+    Equivalent norms: 
+    Finite dimensional spaces:
+    Normes ||· ||p on Rn and Cn: 
+    Banach spaces: 
+    Absolutely convergent series on Banach spaces:
+    Continuous linear functions: 
+    norme of a continuous linear function:
+    Norme of uniform convergence: 
+    Continuous bounded Banach space valued linear functions on a metric space:
+    Compactness of parts of a normed vector space: 
+    Riesz’ Theorem: 
+    d’Ascoli’s Theorem:
+  Hilbert Spaces:
+    Hilbert projection theorem: 
+    Orthogonal projection onto closed vector subspaces:
+    Dual space: 
+    Riesz representation theorem: 
+    l2 and L2 cases: 
+    Hilbert bases (in the separable case): 
+    Examples of trigonometric polynomial bases and orthogonal polynomials: 
+    Lax-Milgram theorem:
+    H10(|0,1|) and its application to the Dirichlet problem in one dimension:
+
+Differential Calculus:
+  Differential Calculus:
+    Differentiable functions on an open subset of Rn: 
+    Differentials (linear tangent functions):
+    Derivatives with respect to a vector:
+    Partial derivatives: 
+    Jacobian matrix: 
+    gradient vector: 
+    Hessian matrix: 
+    Composition of differentiable functions: 
+    Mean value theorem: 
+    Differentiable functions:
+    Functions that can be differentiated k times: 
+    Kth partial derivative: 
+    Inversion of differentiation order: 
+    Taylor’s theorem in differential and integral form:
+    Local study of real valued functions:
+    Series representations: 
+    Local extrema: 
+    concavity of functions on an open convex subset of Rn:
+    Diffeomorphisms: 
+    Inverse function theorem: 
+    Implicit function theorem:
+  Differential equations:
+    differential equations of the form X’ = f(t, X) on I × Ω where I is an open interval in R and Ω is an open subset of Rn: Cauchy-Lipschitz Theorem. Maximal solutions. Grönwall lemma.
```

```suggestion
    differential equations of the form X’ = f(t, X) on I × Ω where I is an open interval in R and Ω is an open subset of Rn&#58; Cauchy-Lipschitz Theorem. Maximal solutions. Grönwall lemma.
```
This escapes the colon so that it's not interpreted as a key-value separator.

---

Nooo! Please don't put html entities. You can put quotes around a string to mark it as a string. Also in this case I don't understand why a colon is used. I think we could skip what's left of the colon and make the rest into several entries.

---

OK, I agree that looks nicer.

---

```suggestion
    differential equations of the form X’ = f(t, X) on I × Ω where I is an open interval in R and Ω is an open subset of Rn
   Cauchy-Lipschitz Theorem
   Maximal solutions
   Grönwall lemma
```

---

I was missing new lines here, which was why there were colons inline. Putting this into several lines I think resolves the issue of using colons.